.github/workflows/main.yml

```diff
@@ -17,16 +85,22 @@ jobs:
         uses: actions/checkout@v2
         with:
           lfs: true
-      - name: Build and test on ${{matrix.os}}
+      - name: Download artifact
+        uses: actions/download-artifact@v2
+        with:
+          name: sdist
```

Comment:
Nitpick: rename the artifacts we upload to `source-distribution`?

Reply:
Will do.